README.md

```diff
@@ -37,33 +37,32 @@ doctrine:
                     engine: MyISAM
 ```
 
-## Browser compatibility
+See the [system settings chapter][2] of the user's manual for further information.
+
+## Browser Compatibility
 
 Chrome, Firefox, Safari 12+, IE 11, Edge 17+, Opera, Chrome for Android, Safari for iOS 11.3+, Samsung Internet 8.2+
 
 ## Installation
 
-See the [installation chapter][2] of the user's manual.
+See the [installation chapter][3] of the user's manual.
 
 ## Documentation
 
- * [User's manual][3]
- * [Developers's manual][4]
- * [Cookbook][5]
- * [API reference][6]
+ * [User Manual][4]
+ * [Developer Documentation][5]
 
 ## License
 
 Contao is licensed under the terms of the LGPLv3.
 
-## Getting support
+## Getting Support
 
-Visit the [support page][7] to learn about the available support options.
+Visit the [support page][6] to learn about the available support options.
 
 [1]: https://contao.org
-[2]: https://docs.contao.org/books/manual/current/en/01-installation/installing-contao.html
-[3]: https://docs.contao.org/books/manual/current/
-[4]: https://docs.contao.org/books/extending-contao4/
-[5]: https://docs.contao.org/books/cookbook/
-[6]: https://docs.contao.org/books/api/
-[7]: https://contao.org/en/support.html
+[2]: https://docs.contao.org/manual/de/system/einstellungen/#config-yml
+[3]: https://docs.contao.org/manual/de/installation/
```

Comment:
This is a problem, because since the README.md is in English, it should redirect to the english version of the manual – which we do not have yet, do we?

Reply:
Indeed. It should not link to the German version.

Reply:
I was well aware of that. I would have changed the link as soon as there is an English version available.

Reply:
So how do we solve this? We could remove the links until the english manual has been published.

Reply:
```suggestion
[2]: https://docs.contao.org/dev/reference/config/
```

Reply:
```suggestion
[2]: https://docs.contao.org/dev/getting-started/initial-setup/
```